apps/gitness/src/components/FileContentViewer.tsx

```diff
@@ -124,6 +124,23 @@ export default function FileContentViewer({ repoContent }: FileContentViewerProp
     )
   }
 
+  const CodeView = () => {
```

Comment:
quick question: how does the code editor handle binaries or too large files?

Reply:
binaries are similar to what you see on github / vs-code, attached a screenshot 
will need to test for long files
@srdjan-harness did you check the code editor for long files?
<img width="1710" alt="Screenshot 2024-10-21 at 12 20 20 AM" src="https://github.com/user-attachments/assets/af9d098e-b5d3-4e21-92c0-6ecf05178332">